### PR TITLE
Update Layout

### PR DESCRIPTION
**Pull request by KaiVolland**
This updates the Design of the demo.

localhost_3000_ 27

### CODE REVIEW

**Comment by KaiVolland** [comment]
@dnlkoch if you think this is `!important` i'll find a way :wink:

**Comment by KaiVolland** [comment]
I removed the Aldrich font from all geostyler components and just left it on the demo specific stuff.
Seems to be a bit clearer this way and without messing up with all antd components.

File: src/App.css [hunk 1]
```diff
@@ -1,42 +1,69 @@
...
+.main-content {
+  color: #162162;
   display: flex;
-  flex-direction: row;
   flex: 1;
 }
 
-.gs-header .App-title {
-  flex-grow: 1;
+.maint-content h2 {
```
Comment (annarieger) [inline src/App.css]: typo? `main-content`?

File: public/index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/logo.png">
-    <link rel="stylesheet" href="%PUBLIC_URL%/style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Aldrich" rel="stylesheet">
```
Comment (marcjansen) [inline public/index.html]: I personally like locally served variants especially of google fonts, Did you tehre is https://google-webfonts-helper.herokuapp.com/fonts/aldrich?subsets=latin which might make it into this PR?

**Pull request merged**